CHANGELOG.rst

```diff
@@ -1,6 +1,17 @@
 Changelog
 =========
 
+v1.6.3
```

Comment:
Should this be a patch bump or a minor bump?

Reply:
I think this is a patch 👍